rpc/src/lib.rs

```diff
@@ -38,3 +38,24 @@ pub mod mayastor {
 
     include!(concat!(env!("OUT_DIR"), "/mayastor.rs"));
 }
+
+#[allow(dead_code)]
+#[allow(clippy::type_complexity)]
+#[allow(clippy::unit_arg)]
+#[allow(clippy::redundant_closure)]
+#[allow(clippy::upper_case_acronyms)]
+pub mod mayastorv1 {
+
+    #[derive(Debug)]
+    pub enum Error {
+        ParseError,
+    }
+
+    impl From<()> for Null {
```

Comment:
what's this all about?

Reply:
it seems very generic so why in this module? :thinking:

Reply:
It has to be, as the you cant impl traits otherwise....

Reply:
> It has to be,

What has to be? :) Which traits?